### PR TITLE
Format Doc Blocks

### LINKED ISSUE #2666: Slim\App docblocks have missing throws annotations
The class `\Slim\App` (3.x) has many methods that may throw exceptions, but their docblocks do not contain corresponding `throws` annotations.

Should we add those annotations or would that break installations?

### PR DESCRIPTION
**Pull request by RyanNerd**
- Add missing `@throws`
  - Closes: #2666

- Format DocBlocks to be cohesive
  - Use imports instead of \ClassName
  - Line return after last `@param` and `@return`

  ```
  use Exception;

  /**
   * @param ...
   *
   * @return ...
   *
   * @throws Exception
   */
  ```

  Closes #2668

### CODE REVIEW

File: Slim/Container.php [hunk 1]
```diff
@@ -121,7 +122,7 @@ public function get($id)
         }
         try {
             return $this->offsetGet($id);
-        } catch (\InvalidArgumentException $exception) {
+        } catch (InvalidArgumentException $exception) {
             if ($this->exceptionThrownByContainer($exception)) {
                 throw new SlimContainerException(
```
Comment (l0gicgate) [inline Slim/Container.php:129]: Missing the throw tag for this exception.

File: Slim/App.php [hunk 2]
```diff
@@ -8,6 +8,7 @@
  */
 namespace Slim;
 
+use Interop\Container\Exception\ContainerException;
```
Comment (l0gicgate) [inline Slim/App.php]: Please order the imports alphabetically and remove any unused imports.

File: Slim/Container.php (hunk 1 above)
Comment (RyanNerd) [reply Slim/Container.php:129]: Fixed in the last commit I just pushed.

File: Slim/App.php (hunk 2 above)
Comment (RyanNerd) [reply Slim/App.php]: This will take me some time to sort out. Right now It's 1:11am my local time. I'll see if tomorrow I can find some time...

Comment (RyanNerd) [reply Slim/App.php]: Latest commit and push sorts all imports alphabetically

**Comment by RyanNerd** [comment]
Travis is reporting: Slim\Tests\AppTest::testRunNotAllowedWithoutHandler
ReflectionException: Class MethodNotAllowedException does not exist.
I've tried several times to fix this (remove import and use inline import for example). I'm stuck not sure why this class is not loading or being detected and loaded.

**Comment by l0gicgate** [comment]
See #2671

**Pull request closed without merging**